### PR TITLE
Update fractional bias and test_geometric_mae to generate data with different rank

### PR DESCRIPTION
**Pull request by puhuk**
Description: Update `fractional bias` and `test_geometric_mae`

Check list:

- [x] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@puhuk PRs title is incorrect and doesn't reflect the content. Please fix it

**Comment by vfdev-5** [comment]
@puhuk CI is failing, please update this PR

**Review by sadra-barikbin — APPROVED** [review]

**Comment by sadra-barikbin** [comment]
@vfdev-5 , `test_geometric_mean_relative_absolute_error.py::test_integration` is failing without any change related to it. Is this a rare case?
image

**Comment by vfdev-5** [comment]
@sadra-barikbin seems like failures are related to this PR and master and others are not failing. We can check `test_geometric_mean_relative_absolute_error.py::test_integration` on `main` with different seed to see if it is related to randomness seed.

File: tests/ignite/contrib/metrics/regression/test_geometric_mean_absolute_error.py [hunk 1]
```diff
@@ -186,7 +187,8 @@ def update(engine, i):
     if device.type != "xla":
         metric_devices.append(idist.device())
     for metric_device in metric_devices:
-        for _ in range(2):
+        for i in range(2):
+            torch.manual_seed(12 + rank + i)
```
Comment (sadra-barikbin) [inline tests/ignite/contrib/metrics/regression/test_geometric_mean_absolute_error.py:191]: @vfdev-5 ,When `12` changed to `11` tests passed.

**Comment by vfdev-5** [comment]
@puhuk what do you suggest to fix the tests and make this PR merged ?

**Comment by puhuk** [comment]
@vfdev-5 as @sadra-barikbin  mentioned, I change the random seed `12` to `11`

**Review by vfdev-5 — APPROVED** [review]
Thanks @puhuk

**Pull request merged**